sr_gui_dynamic_plot_tool/src/sr_gui_dynamic_plot_tool/dynamic_plot_tool.py

```diff
@@ -0,0 +1,467 @@
+#!/usr/bin/env python
+#
+# Copyright 2018 Shadow Robot Company Ltd.
+
+import os
+import rospkg
+import rospy
+
+from xml.etree import ElementTree as ET
+
+from qt_gui.plugin import Plugin
+from python_qt_binding import loadUi
+
+from QtCore import Qt
+import QtWidgets
+from QtWidgets import QWidget, QMessageBox
+from QtGui import QFont
+from sr_utilities.hand_finder import HandFinder
+
+import xml.etree.ElementTree as xmlTool
+
+
+class SrGuiDynamicPlotTool(Plugin):
+    """
+    A rosgui plugin to easily decide which hand's information to plot.
+    """
+
+    def __init__(self, context):
+        super(SrGuiDynamicPlotTool, self).__init__(context)
+        self.setObjectName('SrGuiDynamicPlotTool')
+        self._widget = QWidget()
+        self._script_name = ""
+
+        ui_file = os.path.join(rospkg.RosPack().get_path(
+            'sr_gui_dynamic_plot_tool'), 'uis', 'SrGuiDynamicPlotTool.ui')
+
+        loadUi(ui_file, self._widget)
+
+        self._widget.setObjectName('SrGuiDynamicPlotToolUi')
+        context.add_widget(self._widget)
+
+        self.layout = self._widget.layout()
+
+        self.script_dir = os.path.expanduser("~/projects/shadow_robot/base_deps/src/sr_visualization_common/"
```

Comment:
hardcoded path is not good idea. Maybe better is to use ROS to get path to package and build relative path from there